packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -580,7 +580,7 @@ export const ContentPanel: FC = () => {
                                                                             }
                                                                             style={{
                                                                                 height: inputSectionHeight,
-                                                                                flex: 1,
+                                                                                flex: inputSectionWidth,
```

Comment:
Sometimes the chart wasn't always rendering right. We have a lot of transition animations on this page and it seemed to render halfway through one. This is pretty explicit.